src/setupTests.js

```diff
@@ -1,5 +1,4 @@
-import 'react-app-polyfill/ie11'
-import 'react-app-polyfill/stable'
+import 'whatwg-fetch'
```

Comment:
Both of the react-app-polyfills were detected as leaking by jest. We only needed a `fetch` polyfill.